src/lib/mdns/Resolver.h

```diff
@@ -35,7 +35,50 @@ struct ResolvedNodeData
     Inet::IPAddress mAddress;
     uint16_t mPort;
 };
+struct CommissionableNodeData
+{
+    // TODO(cecille): is 4 OK? IPv6 LL, GUA, ULA, IPv4?
+    static constexpr int kMaxIPAddresses = 5;
+    // Largest host name is 64-bits in hex.
+    static constexpr int kHostNameSize = 16;
+
+    char hostName[kHostNameSize + 1];
+    uint16_t longDiscriminator;
```

Comment:
The `bool additionalPairing` and `bool isInCommissioningMode` are missing (see `AP` and `CM` TXT records in paragraph [4.41] of 0.7-core spec

Reply:
Yup - there's a lot of stuff missing that came in as part of the second ballot. Plan is to add that in a separate PR once this gets in - incremental improvement.